### DISCUSSION TITLE
自动战斗在解析作业后的提示文本是否需要调整位置

### DISCUSSION THREAD

**Discussion by status102** [MaaAssistantArknights/MaaAssistantArknights#6600] [General]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/102887808/db3b397a-820b-4cc5-9082-c8f8caaf189d)

现在新版加完战斗列表之后太长了，完全遮挡了作业信息部分。
或许挪到作业信息之前会好一些？

**Comment by Constrat** [comment]
It would make more sense. We can transfer this to an enhancement issues. Or if you want directly to a PR to switch them.
The instructions should stay still at the top. But when you load a copilot file it should scroll to the first line of the copilot

**Comment by status102** [comment]
I will make a PR to modify it.
